website/source/docs/connect/registration/service-registration.html.md

```diff
@@ -81,7 +81,8 @@ registering a proxy instance.
     "local_service_address": "127.0.0.1",
     "local_service_port": 9090,
     "config": {},
-    "upstreams": []
+    "upstreams": [],
```

Comment:
I can't write a comment any higher up on the doc but we might want to change this doc's name to something like "Sidecar Proxy Standalone Registration" and the other one on this level to "Sidecar Proxy Nested Registration" so that it's clear that this isn't a registration for Gateways.

Reply:
Well technically the "stand alone" one isn't necessarily a sidecar. It could proxy several instances of the same service instead of being a 1-1 association if the `sidecar_service` stanza in a config is used.

Reply:
Ah, interesting. Maybe we can distinguish it some other way... Could it be a bit clearer if we say "Service Proxy Registration" rather than "Proxy Service Registration"?